python/ray/tests/test_global_state.py

```diff
@@ -108,6 +110,27 @@ def get_state():
     assert get_state() == dead_state
 
 
+def test_global_state_worker_table(ray_start_regular):
+
+    # Worker table should be empty at first.
+    assert len(ray.state.workers()) == 0
+
+    # Add global_worker.
+    worker_id = ray.worker.global_worker.worker_id
+    worker_type = gcs_utils.WORKER
+    worker_info = {"key1": "value1"}
+    result = ray.state.state.add_worker(worker_id, worker_type, worker_info)
+
+    # Add worker success.
+    assert result is True
+
+    # Get worker table from gcs.
+    workers_data = ray.state.workers()
+
+    assert len(workers_data) == 1
+    assert workers_data[binary_to_hex(worker_id)] is not None
```

Comment:
```suggestion
    assert binary_to_hex(worker_id) in workers_data
```

Reply:
`workers_data[binary_to_hex(worker_id)]` will throw exception if `binary_to_hex(worker_id)` doesn't exist in `workers_data`

Reply:
updated